packages/graphql/tests/integration/issues/2620.int.test.ts

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Please rename the file to `2630.int.test.ts` since #2630 contains the full bug report.